test/testdata/lsp/bad_field_edits.1.rbupdate.document-symbols.exp

```diff
@@ -34,22 +34,22 @@
                     "kind": 14,
                     "range": {
                         "start": {
-                            "line": 3,
-                            "character": 2
+                            "line": 1,
```

Comment:
The existing exp file is incorrect, and fails now that I've fixed rbupdate tests.

`update_exp_files` generates this exp file, which is correct!

(The old *.exp file highlighted a bug in the reported ranges which has since been fixed.)